realip.go

```diff
@@ -71,7 +71,7 @@ func FromRequest(r *http.Request) string {
 	}
 
 	// Check list of IP in X-Forwarded-For and return the first global address
-	for _, address := range strings.Split(xForwardedFor, ",") {
+	for _, address := range xForwardedFor {
```

Comment:
If `xForwardedFor` is an array, then `address` may be a single IP or a comma delimited string. Suggest something like this:

```
  for _, a := range xForwardedFor {
      for _, b := range string.Split(a, ",") {
        address := strings.TrimSpace(b)
        isPrivate, err := isPrivateAddress(address)
       	  if !isPrivate && err == nil {
            return address
        }
     }
  }
```

Reply:
Ah.. thanks. Individual items may still be comma-delimited.

Reply:
Added the suggestion and also a test for this.